components/PoisCard/PoiCard.vue

```diff
@@ -65,7 +65,7 @@
 
         {{ category }}
       </div>
-
+      <Stars :stars="stars"></Stars>
```

Comment:
Should go here https://github.com/teritorio/vido/blob/feature/clear-all-categories/components/Fields/Field.vue